aws/resource_aws_ssm_maintenance_window_task_test.go

```diff
@@ -141,6 +141,8 @@ resource "aws_ssm_maintenance_window_task" "target" {
   task_type = "RUN_COMMAND"
   task_arn = "AWS-RunShellScript"
   priority = 1
+  name = "TestMaintenanceWindowTask"
```

Comment:
In order to validate that these are actually set and retrieved, we should extend [the related acceptance test](https://github.com/terraform-providers/terraform-provider-aws/blob/2728e9d7828fd75ff07a1d539f8295f7c3fdcf58/aws/resource_aws_ssm_maintenance_window_task_test.go#L27) with something like:

```go
func TestAccAWSSSMMaintenanceWindowTask_basic(t *testing.T) {
	var task ssm.MaintenanceWindowTask

	name := acctest.RandString(10)
	resource.Test(t, resource.TestCase{
		PreCheck:     func() { testAccPreCheck(t) },
		Providers:    testAccProviders,
		CheckDestroy: testAccCheckAWSSSMMaintenanceWindowTaskDestroy,
		Steps: []resource.TestStep{
			{
				Config: testAccAWSSSMMaintenanceWindowTaskBasicConfig(name),
				Check: resource.ComposeTestCheckFunc(
					testAccCheckAWSSSMMaintenanceWindowTaskExists("aws_ssm_maintenance_window_task.target", &task),
resource.TestCheckResourceAttr("aws_ssm_maintenance_window_task.target", "name", "TestMaintenanceWindowTask"),
resource.TestCheckResourceAttr("aws_ssm_maintenance_window_task.target", "description", "This ressource is for test purpose only"),
				),
			},
		},
	})
}
```

Reply:
Seems working with your example @Ninir 
thanks